inclusive-design/wecount.inclusivedesign.ca
# Improve contrast ratio for links (resolves #3)

## Description
Changes colour of links on white backgrounds and yellow background to improve contrast ratio.


## Steps to test

1. Go to the deploy preview
2. Navigate to the Views page
3. Expand Tag Filters and select one or more tags
4. Click "Apply Filters" button
4. Run lighthouse
5. See accessibility section of the lighthouse report
6. Navigate to “Inclusion Challenges” page
7. Run lighthouse
8. See accessibility section of the lighthouse report
9. Repeat steps 7 and 8 for a post page, “Learn” page and "Initiatives" page

**Expected behavior:** 

Should score 100 on accessibility in all lighthouse tests.

## Additional information
N/A


## Related issues


closes #331
